src/classes/RD_RecurringDonations.cls

```diff
@@ -795,277 +721,463 @@ public class RD_RecurringDonations {
     * @return void
     */ 
     public static void handleRecurringDonationUpdate(list<npe03__Recurring_Donation__c> recurringDonations, map<id, npe03__Recurring_Donation__c> oldRecurringDonations, TDTM_Runnable.DmlWrapper dmlWrapper){     
-        //opening a closed recurring donation
-        set<id> reevaluateOpps = new set<id>();
-        list<npe03__Recurring_Donation__c> reinsertOpps = new list<npe03__Recurring_Donation__c>(); 
-        // closed an open recurring donation
-        list<npe03__Recurring_Donation__c> newlycloseddonations = new list<npe03__Recurring_Donation__c>();
-         
+
+        System.debug('+++++Hitting handleRecurringDonationUpdate');
```

Comment:
All `System.debug` statements should be removed. If they're really needed for some level of future debugging, they should be replaced with UTIL_Debug.debug().